burn-book/src/building-blocks/tensor.md

```diff
@@ -172,6 +172,7 @@ Those operations are available for numeric tensor kinds: `Float` and `Int`.
 
 | Burn                                                            | PyTorch Equivalent                             |
 | --------------------------------------------------------------- | ---------------------------------------------- |
+| `Tensor::diagonal(size, device)`                                | N/A                                            |
```

Comment:
Agreed! I didn't want to force the rename for this simple PR, but I found that the name `diagonal` was not as representative. It is more in line with `diag(ones(size))` since it produces an array with ones on the diagonal (aka, the identity matrix but it seems every library calls this `eye` for [**I**]dentity).

Reply:
Done. Renamed it.